### ISSUE TITLE
Changing temp from HA dash board changes for all zones but not specific zone

### ISSUE THREAD

**Issue by lychee128** [dlarrick/hass-kumo#86]
I may be overlooking a setting but I can't seem to change the temperature for my single zone.  I have only 1 unit in 1 zone (Attic), and my site is "Home." On the HA card, it displays the correct settings.  But when I change the temperature in HA, my Kumo App displays this setting "All Zone" but not for "Attic".  On my app, the set temp is displayed as "---" for "Attic" but the Home / All Zones tile shows the correct setting.  Is there a setting I need to add to have the entity set the temp for my zone?   Not sure if I'm explaining clearly, but it seems like HA sets for all the zones.

**Comment by omriasta** [comment]
Is the unit working when you turn it on from home assistant? There is an issue with the conversion from Celsius to Fahrenheit. If the exact value isn't in the table the kumo app shows these dashes but the unit should still work and have the correct temp.

**Comment by omriasta** [comment]
See https://github.com/dlarrick/pykumo/issues/15

**Comment by lychee128** [comment]
Yes, the unit turns on ok. The odd thing is that it seems like it works - it's just that the settings are only displayed in the Home / All Zones tile, and not the specific Attic tile which is where the unit is located.  If I go to the Kumo app and adjust the temp, then it goes from "---" to a number.  And HA reads that correctly.  In HA, there's only 1 entity (labeled "Attic").  But somehow, it seems like the settings don't get synchronized between Kumo Cloud and the HA integration.

**Comment by omriasta** [comment]
They are synced, the -- is just an issue of the kumo app not know what to display... I honestly don't even have the app anymore because it didn't work as well as the integration and I just use home assistant now. I kind of remember there being some sort of ticket with zones as well, you can look at the closed tickets to find it...

**Comment by lychee128** [comment]
Got it.  Actually, I’ve also had a lot of problems with the native Kumo app but the HA integration has been working great so far.  I’m hoping to just use it from now on.

I’ve read through the documentation and wondering if there is any advantage for leaving “prefer local cache” unchecked if I’m using a static IP?

Thanks.

**Comment by dlarrick** [comment]
If you have a static IP I advise turning on "prefer local cache". HA startup will be faster, and you will be more insulated from KumoCloud server outages.

**Comment by lychee128** [comment]
Thanks.  Is there a recommended way of changing this setting?  Should I create a YAML entry, or just delete the integration and start over?

Also, in case my static IP has to change (eg, router reconfiguration), where do I change the IP address?  My IP was found during the integration setup but I can’t seem to find a corresponding YAML entry in any of the custom config files.


Thanks.

**Comment by dlarrick** [comment]
Huh, I thought you could change "prefer cache" from the Configure button on the integration but I'm not seeing it. @omriasta any ideas?
So... yeah, remove / re-add is probably your best bet. You *can* set an updated IP address through Configure, or edit `kumo_cache.json` directly (and restart HA).

**Comment by omriasta** [comment]
I guess I missed that setting when I created the options in the config flow...

**Comment by lychee128** [comment]
Got it, thanks.  It's no problem to remove and re-add — I just wanted to make sure I wasn't missing anything.

One more thing I just noticed...  every so often, the temp setting on my Kumo app is 1-2 degrees (F) off from the setting display in my HA card.  I saw a couple of comments on this related to the syncing between the 2, and that if I use both the settings may not always sync up since there are accessing the same API (so I should pick one).

So I'm assuming just going with HA and doing all my setting there is fine, and there's no reason to ever use the Kumo app again?  The integration works so much better than the Kumo app for me.
